src/components/atoms/Anchor.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "@emotion/styled";
+import theme from "../../utils/theme/theme";
+import {
+  tabletQuery,
+  mobileQuery,
+  desktopQuery,
+  laptopQuery
+} from "../../utils/mediaqueries";
+
+const StyledAnchor = styled.a({
+  color: theme.colors.darkGold,
+  textDecoration: "none",
+  fontWeight: "bold",
```

Comment:
implement a 'bold' prop, like we use in `<Paragraph />`. We don't want every link to be bold